### ISSUE TITLE
early stopping seems not work?

### ISSUE THREAD

**Issue by liuxiaotong15** [materialsvirtuallab/megnet#72]
I'm using megnet in a classification problem, and set patience to 3. The training msg is under:

> Epoch 10/10000
> 16/16 [==============================] - 341s 21s/step - loss: 0.3728 - val_loss: 0.4102
> Epoch 11/10000
> 16/16 [==============================] - 343s 21s/step - loss: 0.3762 - val_loss: 0.3696
> Epoch 12/10000
> 16/16 [==============================] - 344s 22s/step - loss: 0.3728 - val_loss: 0.3583
> Epoch 13/10000
> 16/16 [==============================] - 342s 21s/step - loss: 0.3702 - val_loss: 0.3748
> INFO:megnet.callbacks:val_acc does not improve after 3, stopping the fitting...
> Epoch 14/10000
> 16/16 [==============================] - 339s 21s/step - loss: 0.3723 - val_loss: 0.3999
> INFO:megnet.callbacks:val_acc does not improve after 3, stopping the fitting...
> Epoch 15/10000
> 16/16 [==============================] - 344s 21s/step - loss: 0.3748 - val_loss: 0.3903
> INFO:megnet.callbacks:val_acc does not improve after 3, stopping the fitting...
> Epoch 16/10000
> 16/16 [==============================] - 339s 21s/step - loss: 0.3739 - val_loss: 0.3549
> INFO:megnet.callbacks:val_acc does not improve after 3, stopping the fitting...
> Epoch 17/10000

so stopping didn't stop?
thanks

**Comment by chc273** [comment]
update your megnet to latest megnet repo. It should solve the problem. There are many changes in tensorflow2.0 that broke some previous behavior. It should have been fixed.

**Comment by liuxiaotong15** [comment]
My megnet version is 1.0.2 installed from pip, i found latest release in github is 1.0.1?
This is a different problem with #71 , so I add a new issue. 
Because of #71, I have to run on CPU in only 1000 data training...

**Comment by chc273** [comment]
V1.0.3 is available now.

**Comment by liuxiaotong15** [comment]
ok, thanks. i will try and tell you the results later.

**Comment by liuxiaotong15** [comment]
yes, #72 fixed.
but for #71 , problem still existed.
